chaos_genius/connectors/snowflake.py

```diff
@@ -78,4 +86,11 @@ def get_schema(self):
     def get_schema_names_list(self):
         data = self.inspector.get_schema_names()
         return data
-        
\ No newline at end of file
+        
+    def resolve_identifier(self, identifier: str) -> str:
+        """Convert an identifier to uppercase if it is valid or return the original string."""
+        compare_string = string.ascii_lowercase + "_$"
```

Comment:
@kartikay-bagla Can you add the code level comment for this solution along with the snowflake link if possible?

Reply:
done